README.md

```diff
@@ -1 +1,34 @@
-# info
\ No newline at end of file
+# A simple landing page
+
+> Set up the project, the goal here is to master all of the tools and best practices about Linter and Gitflow.
+
+
+Additional description about the project and its features.
```

Comment:
Hi @sazukeR , great job so far, I suggest please put a screenshot here for the result of your page